vignettes/basic-use.Rmd

```diff
@@ -0,0 +1,94 @@
+---
+title: "Basic Use"
+author: "Mark Ewing"
+date: "2019-04-02"
+output: rmarkdown::html_vignette
+vignette: >
+  %\VignetteIndexEntry{Vignette Title}
+  %\VignetteEngine{knitr::rmarkdown}
+  %\VignetteEncoding{UTF-8}
+---
+
+```{r setup, include = FALSE}
+knitr::opts_chunk$set(
+  collapse = TRUE,
+  comment = "#>"
+)
+```
+
+## Creating a Requirements File
+
+Because a requirements file is a simple text file, It's easy to create.
+
+#### From RStudio
+
+File > New File > Text File
+Then save the file as requirements.txt
+
+#### Linux and Unix-like Systems (e.g. Macs)
+
+touch requirements.txt
+
```

Comment:
Not sure the value of advising users on how to create a `.txt`. I like the introduction of 'it's easy', and then I think immediately getting a few example lines in front of them would be good.  I also think working from a 3-4 line example of requirements would make things more obvious down the line. (i.e. when specifying the structure it could be annotated on the example they just saw)

If for whatever reason they don't know how to create a text file, I think we should point them to the `gen_req` utility.

Reply:
This has been done in 1d42077c9608496abc2100e4255d34f6d5d5b8cf